ui-spa/playwright/e2e/transfer-error-page.spec.ts

```
@@ -0,0 +1,179 @@
+    await expect(page.getByTestId("netapp-table-wrapper")).not.toBeVisible();
+    await expect(
+      page.getByTestId("tab-content-transfer-materials"),
+    ).toContainText("Indexing transfer from egress to shared drive...");
+    await expect(
+      page.getByTestId("tab-content-transfer-materials"),
+    ).toContainText("Completing transfer from egress to shared drive...");
+    await expect(page.getByTestId("transfer-spinner")).not.toBeVisible();
+    await expect(page).toHaveURL("/case/12/case-management/transfer-errors");
+    await expect(page.locator("h1")).toHaveText(
+      "There is a problem transferring files",
+    );
+    const listItems = page.locator("ul > li");
+    await expect(listItems).toHaveCount(2);
+    await expect(listItems).toHaveText([
+      "try again",
+      "check the activity log to see if any files or folders have transferred successfully",
+    ]);
+    await expect(page.getByRole("link", { name: "Back" })).toBeVisible();
+    await expect(page.getByRole("link", { name: "Back" })).toHaveAttribute(
+      "href",
+      "/case/12/case-management",
+    );
+    await expect(page.getByTestId("contact-information")).toHaveText(
+      "To get help, call the Service Desk 0800 692 6996. Tell them you're seeing error code: transfer-id-egress-to-netapp.",
+    );
+    await expect(page.getByRole("button", { name: "Continue" })).toBeVisible();
+    await page.getByRole("button", { name: "Continue" }).click();
+    await expect(page).toHaveURL("/case/12/case-management");
+    await expect(page.locator("h1")).toHaveText(`Thunderstruck`);
+  });
+  test("Should show the transfer error page, if the transfer status is `Failed`", async ({
+    page,
+    worker,
+  }) => {
+    await worker.use(
+      http.get(
+        "https://mocked-out-api/api/v1/filetransfer/transfer-id-egress-to-netapp/status",
+        async () => {
+          await delay(10);
+          return HttpResponse.json({
+            status: "PartiallyCompleted",
```

Great work on covering the error page scenario! I noticed that in this test for the `Failed `status, the mock response is still using `status: "PartiallyCompleted"`. For accuracy and to ensure we’re testing the correct flow, could you update this to `status: "Failed"`? This will make sure the test truly reflects the intended scenario. Thanks!